src/components/learner-credit-management/BudgetAssignmentsTable.jsx

```diff
@@ -112,16 +119,16 @@ const BudgetAssignmentsTable = ({
           disableFilters: true,
         },
       ]}
-      additionalColumns={[
+      additionalColumns={(!isRetired) ? [
         {
           Header: '',
           Cell: AssignmentRowActionTableCell,
           id: 'action',
         },
-      ]}
-      tableActions={[
+      ] : []}
```

Comment:
[nit] This logic should be resolved outside of the component above the return statement